extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeAbbreviationSubCountMap.java

```diff
@@ -0,0 +1,136 @@
+package org.schabi.newpipe.extractor.services.youtube.extractors;
+
+import java.util.HashMap;
+
+public class YoutubeAbbreviationSubCountMap {
```

Comment:
Can you please add a JavaDoc for this class?

Reply:
I am not sure about the class name, too. I assume that these abbreviations can be used by other services, too. Is this correct? If yes, please move this file to the extractor utils.

Reply:
Yes, it can be used if one wants to parse abbreviations from languages (and if some are missing for a service, it can easily be added).

Reply:
Is the JavaDoc good?

Reply:
yes, looks good. however, the class is not a map. we should rename it and change that part in the doc, too.